client/multiplexers.go

```diff
@@ -59,13 +59,15 @@ func ServiceAccountRegionScopeMultiplexer(service string) func(meta schema.Clien
 		var l = make([]schema.ClientMeta, 0)
 		client := meta.(*Client)
 		for accountID := range client.ServicesManager.services {
-			// always fetch cloudfront related resources
-			l = append(l, client.withAccountIDRegionAndScope(accountID, "us-east-1", wafv2types.ScopeCloudfront))
 			for region := range client.ServicesManager.services[accountID] {
 				if !isSupportedServiceForRegion(service, region) {
 					meta.Logger().Trace("region is not supported for service", "service", service, "region", region)
 					continue
 				}
+				if region == "us-east-1" {
+					// fetch cloudfront related resources only when us-east-1 is enabled
+					l = append(l, client.withAccountIDRegionAndScope(accountID, "us-east-1", wafv2types.ScopeCloudfront))
+				}
```

Comment:
Its not possible to disable `us-east-1` it is only possible to disable regions brought online after March 20, 2019:

```
Regions introduced before March 20, 2019 are enabled by default. You can begin creating and managing resources in these Regions immediately. You cannot enable or disable a Region that is enabled by default.
```

Reply:
"selected" is probably a better word. I meant like "enabled for fetching".

Reply:
I would argue that if a customer wants to grab WAF resources that should outweigh their specification of what regions to grab from. This is the same behavior that S3 has

Reply:
Reverted to older (desired) behavior of always fetching cloudfront scope